net/freeradius/src/opnsense/service/templates/OPNsense/Freeradius/users

```diff
@@ -15,7 +15,8 @@
        WISPr-Bandwidth-Min-Down = {{ user_list.wispr_bw_min_down }}{% endif %}{% if user_list.wispr_bw_max_down is defined %},
        WISPr-Bandwidth-Max-Down = {{ user_list.wispr_bw_max_down }}{% endif %}{% if user_list.chillispot_bw_max_up is defined %},
        ChilliSpot-Bandwidth-Max-Up = {{ user_list.chillispot_bw_max_up }}{% endif %}{% if user_list.chillispot_bw_max_down is defined %},
-       ChilliSpot-Bandwidth-Max-Down = {{ user_list.chillispot_bw_max_down }}
+       ChilliSpot-Bandwidth-Max-Down = {{ user_list.chillispot_bw_max_down }}{% endif %}{% if user_list.sessionlimit_max_session_limit is defined %},
+       Max-Daily-Session := {{ user_list.sessionlimit_max_session_limit }}
```

Comment:
I mean this line is not guarded.

Reply:
Hm, I don't get it, it's guarded by the if above at the end of the line. It's ugly to read but I need it this  way cause of the comma at the end.

Reply:
Ok, hard to read but I see it now. Please disregard.